java/pom.xml

```diff
@@ -77,6 +78,7 @@
         <hadoop.version>3.3.3</hadoop.version>
         <elasticsearch.version>7.17.14</elasticsearch.version>
         <datastax.version>4.15.0</datastax.version>
+        <flink.version>1.16.0</flink.version>
```

Comment:
1.16.0 seems too old. Can we try 1.18.0 without changing too much code in current PR?